assets/js/googlesitekit-dashboard-details.js

```diff
@@ -48,8 +51,21 @@ domReady( () => {
 	);
 
 	if ( renderTarget ) {
+		// Using global preloaded data since we cannot use selectors
```

Comment:
This doesn't seem to follow the implementation stated in the IB. From reading the comments in the issue, it seems you and @eugene-manuilov had a consensus about using the `useSelect` and data store, as well as moving things around to support that. Was that not possible?

Reply:
Yes, unfortunately, it didn't work as we expected, @kuasha420. That's why Jimmy had to use the preloaded data and direct access to the global variables.

Reply:
Thanks for clarifying @eugene-manuilov ! Makes sense.